Tests/Microsoft.AppCenter.Analytics.Test.Windows/SessionTrackerTest.cs

```diff
@@ -35,6 +35,70 @@ public void InitializeSessionTrackerTest()
             _sessionTracker = new SessionTracker(_mockChannelGroup.Object, _mockChannel.Object);
         }
 
+        /// <summary>
+        /// Verify start session when autogenerate session is disabled.
```

Comment:
```suggestion
        /// Verify start session when manual session tracker is enabled.
```

Reply:
I think the test method is not named properly. Maybe `StartSessionWhenManualSessionTrackerIsEnabled`?